utils/dataset.py

```diff
@@ -463,14 +477,27 @@ def __init__(self, args, path, config):
         )
 
         # depth parameters
-        self.has_depth = False
-        self.depth_scale = None
+        self.has_depth = True
```

Comment:
This hardcoded flag assumes the depth input is always available, but we want to adjust it depending on the sensor_type in config file (either monocular or depth).

Reply:
This comes from the yaml file now